include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
I'm not new to collaboration. I just see any value in GLUI if it cannot undergo a meaningful metamorphosis. If it's impossible for code to "develop" because it must be "maintained" then I think maintainers are like the troll on a bridge, not helping anybody. If my brush with GLUI doesn't go well I will develop it alone and call it COLLADA-UI or something, and it will most likely die here.